### PR TITLE
Feature Show Reminder before answer (#3064)

### PR DESCRIPTION
**Pull request by Loudwig**
In the deck config you can choose to show a reminder instead of showing directly the answer at the end of the autoupdate time.

See [here](https://github.com/ankitects/anki/issues/3064) the discussion about this issue.

### CODE REVIEW

File: ftl/core/deck-config.ftl [hunk 1]
```diff
@@ -448,7 +448,10 @@ deck-config-answer-hard = Answer Hard
 deck-config-answer-good = Answer Good
 deck-config-days-to-simulate = Days to simulate
 deck-config-desired-retention-below-optimal = Your desired retention is below optimal. Increasing it is recommended.
-
+deck-config-after-question-show-answer = Show answer
+deck-config-after-question-show-reminder = Show reminder
```
Comment (dae) [inline ftl/core/deck-config.ftl:452]: Please use Title Case to match the other dropdowns.

File: proto/anki/deck_config.proto [hunk 2]
```diff
@@ -101,6 +101,11 @@ message DeckConfig {
       ANSWER_ACTION_SHOW_REMINDER = 4;
     }
 
+    enum AfterQuestion{
```
Comment (dae) [inline proto/anki/deck_config.proto:104]:
```suggestion
    enum AfterQuestion {
```

Comment (dae) [reply proto/anki/deck_config.proto:104]: If renaming the text to "Question action", please rename the enum too.

**Comment by dae** [comment]
Regarding your question about merging on the associated issue ticket, please merge the latest main changes into your PR branch. You may need to handle some merge conflicts, as there have been recent changes on the main branch to things like ts paths (eg ../components might become $lib/components, etc)

**Comment by dae** [comment]
You shouldn't need to close the PR - just (force)push your changes once you've made them.

**Comment by Loudwig** [comment]
I believe it's good now. I did `./ninja format` and `./ninja check` this time :). Should I try to rebase all of my commit into one or it's good @dae ? (Thanks for your patience).

File: ftl/core/deck-config.ftl [hunk 3]
```diff
@@ -448,7 +448,10 @@ deck-config-answer-hard = Answer Hard
 deck-config-answer-good = Answer Good
 deck-config-days-to-simulate = Days to simulate
 deck-config-desired-retention-below-optimal = Your desired retention is below optimal. Increasing it is recommended.
-
+deck-config-question-action-show-answer = Show Answer
+deck-config-question-action-show-reminder = Show Reminder
+deck-config-question-action = Question action 
+deck-config-question-action-tool-tip = The action to perform after the end of the time of the question. Either show answer and auto update or just show a reminder.
```
Comment (dae) [inline ftl/core/deck-config.ftl:454]: Let's drop the second sentence, as it simply repeats the dropdown options. Suggested wording:

The action to perform after the question is shown, and time has elapsed.

**Review by dae — COMMENTED** [review]
Thank you - almost there!

Two remaining points:

- let's put question action before answer action
- please remove the core-repo change from this PR; that repo will be updated automatically when I next sync translations

**Comment by dae** [comment]
> Should I try to rebase all of my commit into one 

No need, that will happen when I squash-merge the PR.

**Comment by Loudwig** [comment]
while trying to remove core-repo and qt-repo from the PR i did some serious damage and i have strong difficulties going back. I figured that ftl/core-repo and ftl/qt-repo where updtated with the first merge commit. So i tried to revert it, then i updated the file of deck-config but it seems i did not do it properly...@dae

**Comment by dae** [comment]
When you get stuck with git, sometimes the easiest solution is to copy the files you have changed, delete the branch you made, and make it again from the current main branch. Then you can copy the modified files in, and check nothing else was changed except the changes you made.

**Comment by Loudwig** [comment]
It is in deed faster than resolving the git issues that i had :)@dae.

File: ftl/core/deck-config.ftl [hunk 4]
```diff
@@ -237,6 +237,10 @@ deck-config-seconds-to-show-question = Seconds to show question for
 deck-config-seconds-to-show-question-tooltip-2 = When auto advance is activated, the number of seconds to wait before revealing the answer. Set to 0 to disable.
 deck-config-seconds-to-show-answer = Seconds to show answer for
 deck-config-seconds-to-show-answer-tooltip-2 = When auto advance is activated, the number of seconds to wait before applying the answer action. Set to 0 to disable.
+deck-config-question-action-show-answer = Show Answer
+deck-config-question-action-show-reminder = Show Reminder
+deck-config-question-action = Question action 
+deck-config-question-action-tool-tip = The action to perform after the end of the time of the question.
```
Comment (dae) [inline ftl/core/deck-config.ftl:243]:
```suggestion
deck-config-question-action-tool-tip = The action to perform after the question is shown, and time has elapsed.
```

**Comment by dae** [comment]
Thank you!

**Pull request merged**